R/mozaiekberekentool.R

```diff
@@ -0,0 +1,158 @@
+#' Calculate mosaic percentage
+#' 
+#' Takes a experiment number, gender, snp dataframe and events dataframe
+#' and calculates the percentage of mosaic dna
+#' 
+#' @param exp.nr numbarical experiment number
+#' @param gender string denoting gender
+#'  ( "Male", "Female" or "Unknown")
+#' @param snpm.data dataframe containing snp data
+#'  ( "Chr", "Position", "B Allele Freq")
+#' @param deviations dataframe containing deviations
+#' ( "Chromosome Region",	"Event",	"Length",	"Cytoband",	precentage of CNV Overlap",
+#' 	"Probe Median",	"precentage Heterozygous",	"Probes",	"Count of Gene Symbols")
+#' @param outfile optional path to direct pdf output to, if not set './Rplots.pdf' is used
+#' @import stats
+#' @import grDevices
+#' @import grid
+#' @import gridExtra
+#' @importFrom utils read.table setTxtProgressBar txtProgressBar
+#' @importFrom graphics plot text
+#' @export
+#'
+#Versionnumber: 0.6.0.0
+MosaicCalculator <- function(exp.nr, gender, snpm.data, deviations, outfile){
+  print("Dank u voor het kiezen van Mozaiek Bereken Tool, ontwikkeld door Robert Sietsma")
+  #Set the option of the output decimal to comma
+  options(OutDec = ",")
```

Comment:
I think for international purposes we want to use a . as separator for decimals?

Reply:
I don't know, i suggest we check with Marloes and Lennart, i suggest to leave is as is for now until we know what the users would like, i also do not know is the string processing in the script depends of the output decimal being an comma.

Reply:
Agree